src/components/CouponDetails/index.jsx

```diff
@@ -569,12 +573,12 @@ class CouponDetails extends React.Component {
   }
 
   renderSuccessMessage({ title, message }) {
-    const { hasError } = this.props;
+    const { hasError, couponOverviewError } = this.props;
```

Comment:
I think you said you had an update in mind for this, but wanted to drop a note here that the variable names are potentially a tad confusing here 👍

Reply:
Yeah, I realized the names are potentially confusing but am opting to address renaming them to be clearer in a subsequent PR.